Android.gitignore

```diff
@@ -34,3 +34,7 @@ captures/
 
 # Intellij
 *.iml
+.idea
+
+# Only needed under Mac OS
```

Comment:
We track this as well as other OSX specific rules under `Global/OSX.gitignore` so things are more composable